docs/reference/submission_schemas/v1.rst

```diff
@@ -0,0 +1,8 @@
+Version 1
+=========
+
+This document describes the schema of the ``test_submission_schema/v1.json`` files as part of the submission. To get the latest JSON schema file, go to the Test Submission Schema repository.
+        
+.. jsonschema:: ../../../test_submission_schema/v1.json
```

Comment:
Hey @tang-mm , I received an error saying that the Dependency file <path/to/json> does not exist, but the relative path looks good to me, can you have a look into this? Thanks.